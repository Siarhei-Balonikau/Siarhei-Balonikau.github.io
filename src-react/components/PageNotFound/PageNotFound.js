import React from 'react';
import PropTypes from 'prop-types';


const PageNotFound = props => {
  return (
    <div>
      Page not found
    </div>
  );
};

export default PageNotFound;