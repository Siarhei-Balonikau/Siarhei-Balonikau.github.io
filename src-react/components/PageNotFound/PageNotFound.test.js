import React, { Component } from 'react';
import { shallow } from 'enzyme';
import PageNotFound from './PageNotFound'; 
import toJson from 'enzyme-to-json'

describe('subscribe component', () => {
  let wrapper, props;

  beforeEach(() => {
    props = {
      
    };
  
    wrapper = shallow(<PageNotFound {...props} />);
  });
  
  it('should render component', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});