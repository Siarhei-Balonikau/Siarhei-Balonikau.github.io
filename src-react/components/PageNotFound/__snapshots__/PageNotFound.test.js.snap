// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`subscribe component should render component 1`] = `
<div>
  Page not found
</div>
`;
